fatxxj/TestZaAud
# Add file

## Pull request
Related to ticket # 12345

Changes:
- Added file text.txt to show how PR/MRs work

How to verify: 
- merge the pull request and execute git pull locally and check if test.txt file exists.
